groovy/postInit/progression/Batteries.groovy

```diff
@@ -61,6 +61,10 @@ crafting.addShaped("drum_lead", metaitem('drum.lead'), [
 crafting.addShapeless("drum_nbt_lead", metaitem('drum.lead'), [
         metaitem('drum.lead').noreturn()
 ]);
+crafting.addShapeless("drum_nbt_brass", metaitem('drum.brass'), [
```

Comment:
Why is this recipe logic done in batteries?

Reply:
Someone aleady made the temporary fix for the lead drum here, so I just copied their code and changed the lead to brass.

I have no idea who put the lead drum clearing recipe first.